core/client/fs/src/main/java/alluxio/client/block/stream/BlockInStream.java

```diff
@@ -100,18 +99,9 @@
   public static BlockInStream create(FileSystemContext context, BlockInfo info,
       WorkerNetAddress dataSource, BlockInStreamSource dataSourceType, InStreamOptions options)
       throws IOException {
-    URIStatus status = options.getStatus();
-    ReadType readType = ReadType.fromProto(options.getOptions().getReadType());
-
     long blockId = info.getBlockId();
     long blockSize = info.getLength();
 
-    // Construct the partial read request
-    ReadRequest.Builder builder =
-        ReadRequest.newBuilder().setBlockId(blockId).setPromote(readType.isPromote());
-    // Add UFS fallback options
-    builder.setOpenUfsBlockOptions(options.getOpenUfsBlockOptions(blockId));
-    builder.setPositionShort(options.getPositionShort());
```

Comment:
moved to `createGrpcBlockInStream`. For other ways of creating `BlockInStream`, we don't need to build `ReadRequest`